src/__tests__/make-unbuffered-no-cache-request.test.js

```diff
@@ -18,7 +18,7 @@ describe("#makeUnbufferedNoCacheRequest", () => {
     it("should create an agent for the request", () => {
         // Arrange
         const fakeSuperagent = {
-            get: jest.fn().mockReturnThis(),
+            agent: jest.fn().mockReturnThis(),
```

Comment:
We call `get` first now, which means we have to mock it at the import level. And vice versa, `agent` is called from the `get` result. Therefore, in all these tests we have swapped them over.